Add About us page

clytaemnestra opened the pull request.

clytaemnestra committed:
Add About us page

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

europython-society commented:
# Preview available
|Key|Value|
|---|---|
|url|https://ep2025-about-us-2.ep-preview.click|
|last update|2025-05-14T22:16:36.135Z|

clytaemnestra committed:
Update links

clytaemnestra committed:
Merge branch 'ep2025-about-us-2' of https://github.com/EuroPython/website into ep2025-about-us-2

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

nikoshell reviewed (commented):
I suggest to use md in some case can be mdx, if it must be mdx pls use / create astro components instead of huge html block. If html source is bigger then md, we should create astro page.

Other way it will be hard to maintain.

In this case with this content md file will be good.

nikoshell committed:
Convert mdx content to md.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

nikoshell committed:
Merge branch 'ep2025' into ep2025-about-us-2

nikoshell committed:
Update footer links.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

nikoshell committed:
Revert "[pre-commit.ci] auto fixes from pre-commit.com hooks"

This reverts commit f8122d6a95171ece9104e5104ae34212b82bf3d8.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

hypha commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,118 @@
+- Mia Bajić (Vice Chair)
+- Shekhar Koirala
+
+## Where Does Our Profit Go?
```
I think surplus is better than profit.

hypha commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,118 @@
+
+# About Us
+
+The EuroPython conference is organized by volunteers from the Python community
```
The rest of the website is using UK spelling. Might be better to be consistent, i.e organization -> organisation

nikoshell committed:
Revert "[pre-commit.ci] auto fixes from pre-commit.com hooks"

This reverts commit 5a60849b43049bd1fd399e26f329a6a2bfe67bcc.

nikoshell committed:
Reapply "[pre-commit.ci] auto fixes from pre-commit.com hooks"

This reverts commit 3bbf150ca2bfa49fe02f7ceefed6fffd8e2e26e0.

nikoshell committed:
Revert "[pre-commit.ci] auto fixes from pre-commit.com hooks"

This reverts commit f8122d6a95171ece9104e5104ae34212b82bf3d8.

nikoshell committed:
Revert "Update footer links."

This reverts commit 95a6d0f1d15eff9e9a76fcdb7b2a8675191d1f91.

hypha commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,118 @@
+The funds are allocated as follows:
+
+- Grants – We support Python community events across Europe through our grants
+  program. Learn more about grants
```
UK spelling nudge again: program -> programme

hypha commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,118 @@
+- Yuliia Barabash
+
+### 🎨 Communications & Design
+
```
I think Panagiotis and Adriana should be mentioned here.

hypha commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,118 @@
+
+- Artur Czepiel
+- Doreen Peace Nangira Wanyama
+- Koti Vellanki
```
I think our Shekhar needs to be here.

hypha commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,118 @@
+
+- Cheuk Ting Ho
+- Jakub Vysoký
+- Lumír Balhar
```
What about Naomi?

hypha reviewed (commented):
I think we should add the infra team too too and have special mention of people like Ege and Marcin.

clytaemnestra committed:
Add more people

clytaemnestra committed:
Merge branch 'ep2025' into ep2025-about-us-2

clytaemnestra commented:
Hey @hypha thanks a lot for the feedback! The PR is still a draft, I just copied names from the onboarding spreadsheet, so I'm aware that I was missing some people. I've addressed all your remarks and hopefully, I didn't miss anyone 🙂

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

thepetk commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,131 @@
+- Kshitijaa Jaglan
+- Marcin Wierzbanowski
+- Mia Bajić
+- Oleh Kostromin
```
```suggestion
- Oleh Kostromin
- Raquel Dou
```

thepetk replied:
regardless of the incorrect animal facts <3 @hypha

clytaemnestra committed:
Update src/content/pages/about.md

Co-authored-by: Theofanis Petkos <thepetk@gmail.com>

thepetk commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,132 @@
+The EuroPython conference is organised by volunteers from the Python community
+across Europe and beyond.
+
+Behind the scenes, it is run by the EuroPython Society (EPS), a Swedish
```
```suggestion
Behind the scenes, it is run by the [EuroPython Society (EPS)](https://europython-society.org/), a Swedish
```

clytaemnestra committed:
Update src/content/pages/about.md

Co-authored-by: Theofanis Petkos <thepetk@gmail.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

clytaemnestra committed:
Merge branch 'ep2025' into ep2025-about-us-2

egeakman commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,133 @@
+- Cyril Bitterlich
+- Ege Akman
+- Mia Bajić
+- Marcin Wierzbanowski
```
```suggestion
- Marcin Wierzbanowski
- Raquel Dou
```

egeakman commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,133 @@
+conference, as well as overall coordination.
+
+In accordance with Swedish bylaws, the Society is governed by a board of seven
+people, elected annually by its members.
```
```suggestion
In accordance with the [EPS bylaws](https://www.europython-society.org/bylaws/),
the society is governed by a board consisting of a chairperson, a vice chairperson,
and between two and seven additional board members, all elected annually by its
members.
```

egeakman reviewed (commented):
Thanks Mia! A few suggestions:

clytaemnestra committed:
Update src/content/pages/about.md

Co-authored-by: Ege Akman <me@egeakman.dev>

clytaemnestra committed:
Update src/content/pages/about.md

Co-authored-by: Ege Akman <me@egeakman.dev>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

clytaemnestra committed:
Merge branch 'ep2025' into ep2025-about-us-2

clytaemnestra committed:
Merge branch 'ep2025' into ep2025-about-us-2

hypha commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,136 @@
+
+- Joana Owusu-Appiah
+- Naa Ashiorkor Nortey
+
```
the mentor of Mentorship :face_holding_back_tears: 
```suggestion
Theofanis Petkos

```

hypha reviewed (commented):
One wee suggestion :)

clytaemnestra committed:
Update src/content/pages/about.md

Co-authored-by: hypha <ramble.dalloway@gmail.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

clytaemnestra commented:
alright, thank you everyone for your feedback, I've addressed all the notes - can I merge the PR?

clytaemnestra committed:
Merge branch 'ep2025' into ep2025-about-us-2

thepetk commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,136 @@
+### 🗣️ Speaker Mentorship
+
+- Joana Owusu-Appiah
+- Naa Ashiorkor Nortey Theofanis Petkos
```
nit:
```suggestion
- Naa Ashiorkor Nortey
- Theofanis Petkos
```

thepetk commented on src/content/pages/about.md:
```diff
@@ -0,0 +1,136 @@
+- Daksh P. Jain
+- Mia Bajić
+- Panagiotis Kyrillos
+- Tim Hobbs
```
based on the people's celebration channel I think we only miss San!
```suggestion
- Sangharshanan Veera
- Tim Hobbs
```

hypha committed:
Update src/content/pages/about.md

this was my bad!

Co-authored-by: Theofanis Petkos <thepetk@gmail.com>

hypha reviewed (approved):
<3 to all

clytaemnestra committed:
Update src/content/pages/about.md

Co-authored-by: Theofanis Petkos <thepetk@gmail.com>
